components/js/slickGrid.ts

```diff
@@ -543,6 +549,7 @@ export class SlickGrid implements OnChanges, OnInit, OnDestroy, AfterViewInit {
                 editor.setValue(oldValue);
             }
         }
+        this.onRangeRenderCompleted.emit({ startRow: startIndex, endRow: startIndex + count });
```

Comment:
This name seems a bit miss leading, there isn't really when a range is rendered, this is when we have completed getting the data in the range and we are simply informing slick grid of this fact. Is this what you intend for this event to be?

For example when the grid first renders we will have rows 0-50 in 1 window and 50-100 in another window. However slickgrid will have only rendered the visible rows, lets say 0-10. The user then scrolls down to show 20-50. Slickgrid will then render rows 20-50 but this function won't be called on those rows since we already had rows 20-50 in a window.